WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -278,6 +278,20 @@ def update_events
     end
   end
 
+  def update_wcif_events
```

Comment:
Ok! I'll go with the "clumsy, but correct" solution for now.

It may be worth noting that in my upcoming UI changes to start using this `update_events_from_wcif` method, we'll get rid of the `update_events` method. (However, even once the name `update_events` is free, I still like the idea of WCIF being in the name here somewhere, though).